app/Http/Controllers/Admin/KitchenController.php

```diff
@@ -100,4 +100,12 @@ public function update(Request $request, Kitchen $kitchen) {
 	public function destroy(Kitchen $kitchen) {
 		//
 	}
+
+	public function fields(){
+        $fields  = Kitchen::fields()->sortBy('name');
```

Comment:
I don't think we should sort them by name - most likely we want to add a "order" column - so they can also reorder them later

Reply:
cool, will give them order